geode-redis/src/integrationTest/java/org/apache/geode/redis/StringsIntegrationTest.java

```diff
@@ -281,7 +281,7 @@ public void testGetSet_shouldBeAtomic()
 
     latch.countDown();
 
-    Integer getSetSum = future2.get(5, TimeUnit.SECONDS);
+    Integer getSetSum = future2.get(60, TimeUnit.SECONDS);
```

Comment:
Rather than hard-coding a wait limit, most Geode tests use `GeodeAwaitility`. The advantage of the latter is that we can (and have) set a global default maximum timeout (currently 5 minutes).

Reply:
Aha! The part of the code we're working on is old enough that I don't think we've seen an example of that. Thanks for the tip - we'll adopt that going forward.